test/arcade/potts/agent/cell/PottsCellContainerTest.java

```diff
@@ -109,6 +109,8 @@ public void convert_noRegions_createsObject() {
         double criticalVolume = randomDoubleBetween(10, 100);
         double criticalHeight = randomDoubleBetween(10, 100);
         MiniBox parameters = mock(MiniBox.class);
+        // Set up the mock to return "stem" when get("CLASS") is called
+        when(parameters.get("CLASS")).thenReturn("stem");
```

Comment:
I would import the `DEFAULT_CELL_CLASS` from `PottsSeries` and use it here OR have it return an empty string (which should be caught in the default case).